guide/Why_community.md

```diff
@@ -0,0 +1,15 @@
+## Why invest in community
+
+ We recognize that the current business climate is about quicker and quicker releases. However, this can lead to an environment where short term thinking becomes the new normal.
+
+TELUS leadership believes that sometimes you have to slow down to go faster, and that the Design System is one of those initiatives. They are asking you to invest up front in building Community components, knowing that later on the rewards to the entire TELUS ecosystem will be much greater.
+
+**Platform thinking**
+
+The secret to platform thinking is asking ‘Could anyone else benefit from this?’. If the answer is yes, then [submit an issue to TDS Community](https://docs.google.com/presentation/d/1IedSwywxFQKwf9XfP3TcOneQwzILqPSW1WMqVCIZZKw/edit#slide=id.g3ae2c0916b_0_169).
```

Comment:
Does this linked content duplicate https://github.com/telus/tds-community/pull/42 ?
Also, the `tds-community` repo is public facing. Should we point to the resource added in https://github.com/telus/tds-community/pull/42 instead of Google Docs?